modules/specialized-guides/pages/public-cloud-guide/faq/faq-billing-general.adoc

```diff
@@ -56,7 +56,7 @@ Using {productname} with 10 instances registered for management for 3 weeks in t
 Transient model::
 A temporary deployment of {productname} with 20 managed instances, for 2 weeks (336 hours)
 
-The average node count is calculated by adding the number of active nodes (counted hourly) and dividing by the number of hours the managed nodes have been active in the billing cycle.
+The total is calculated by counting number of active nodes (checked hourly). There is a monthly charge for each managed node under management in the billing cycle.
```

Comment:
```suggestion
The total is calculated by counting number of active nodes (checked hourly).
There is a monthly charge for each managed node under management in the billing cycle.
```
We have the rule to start every sentence on a line of its own.  This helps with creating templates for the translation process.